tests/test_endpoints.py

```diff
@@ -435,3 +439,128 @@ def test_agent_capabilities(self):
 
         caps = ca.capabilities
         self.assertEqual(caps['foo'], 1)
+
+
+class TestServices(EndpointTestCase):
+
+    def test_statistics(self):
+
+        stat_data = json_fixture(response_data={
+            "statistics": {
+                "service": [
+                    { "queued": "0" },
+                    { "queued": "1" }
+                ]
+            }
+        })
+
+        with HTTMock(stat_data):
+            stats = ServicesEndpoint.statistics(self.c)
+            self.assertTrue(isinstance(stats, dict))
+            stats_obj = self.c.statistics()
+            self.assertTrue(isinstance(stats_obj, ServiceStatistics))
+
+    def test_statistics_obj(self):
+        stats_obj = ServiceStatistics({
+            'statistics': {
+                'service': [
+                    {
+                        'queued': 0,
+                        'running': 1,
+                        'serviceRegistration': {
+                            'type': 'foo',
+                            'host': 'http://foo'
+                        }
+                    },
+                    {
+                        'queued': 0,
+                        'running': 1,
+                        'serviceRegistration': {
+                            'type': 'fud',
+                            'host': 'http://foo'
+                        }
+                    },
+                    {
+                        'queued': 1,
+                        'running': 2,
+                        'serviceRegistration': {
+                            'type': 'bar',
+                            'host': 'http://bar'
+                        }
+                    },
+                    {
+                        'queued': 0,
+                        'running': 0,
+                        'serviceRegistration': {
+                            'type': 'baz',
+                            'host': 'http://baz'
+                        }
+                    },
+                    {
+                        'queued': 1,
+                        'running': 1,
+                        'serviceRegistration': {
+                            'type': 'baz',
+                            'host': 'http://baz'
+                        }
+                    }
+                ]
+            }
+        }, self.c)
+        self.assertTrue(isinstance(stats_obj.services, list))
```

Comment:
I guess I usually start by writing the simplest thing that could fail. I like having a test fail because the assertion fails and not because of a runtime issue, e.g. trying to access a list item from something that's not a list. That said, definitely not a high value test.